README.rst

```diff
@@ -801,6 +801,29 @@ There's a couple of things to keep in mind when enabling cors for a view:
   requests and matches the ``Origin`` header against a whitelist of origins.
   If the match is successful then return just their ``Origin`` back to them
   in the ``Access-Control-Allow-Origin`` header.
+
+  Example:
+  
+.. code-block:: python
+
+    from chalice import Chalice, Response
+    app = Chalice(app_name='multipleorigincors')
+ 
+    @app.route('/cors_multiple_origins', methods=['GET'])
+    def supports_cors_multiple_origins():
+        origin = app.current_request.to_dict()['headers']['origin']
+        if origin == 'http://allowed1.example.com':
+            return Response(body='You sent a whitelisted origin!',
+                status_code=200,
+                headers={'Access-Control-Allow-Origin': 'http://allowed1.example.com'})
+        elif origin == 'http://allowed2.example.com':
```

Comment:
This is mostly a repeat of the above if, this can be factored out to be a lot clearer.